src/main/java/duke/DateAndTime.java

```diff
@@ -0,0 +1,70 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+
+/**
+ * Manages the date and time based on a user input.
+ */
+public class DateAndTime {
+
+    /**
+     * Parses a given string date. For dates only.
+     * @param date The date in YYYY-MM-DD format
+     * @return The formatted date string
+     */
+    public String dayParse(String date) {
+        int year = LocalDate.parse(date).getYear();
+
+        String month = LocalDate.parse(date).getMonth().toString();
+
+        int day = LocalDate.parse(date).getDayOfMonth();
+
+        return month + " " + day + ", " + year;
+    }
+
+    /**
+     * Formats the date and time
+     * @param date The date in YYYY-MM-DD format.
+     * @param timing The time in HH:MM format
+     * @return The formatted date and time String.
+     */
+    public String dayParse(String date, String timing) {
```

Comment:
If I'm not wrong, method names should be verbs? So perhaps this could be renamed as "parseDay" instead?

Reply:
True! Thanks for the feedback!